src/components/AccountWarning/AccountWarning.tsx

```diff
@@ -146,7 +160,7 @@ function WarningModal(props: WarningModalProps): JSX.Element {
                 if (Date.now() - now > BUTTON_COUNTDOWN_TIME) {
                     clearInterval(interval);
                 }
-            }, 100);
+            }, 1000);
         }
```

Comment:
I realize this this timer wasn't added in this PR, but you'll almost always want a cleanup function in `useEffect()`.  Otherwise, if props.warning changes, you'll have multiple timers running at the same time that interfere with eachother.